packages/edit-site/src/components/layout/router.js

```diff
@@ -179,14 +174,14 @@ export default function useLayoutAreas() {
 	}
 
 	// Navigation
-	if ( path === '/navigation' ) {
+	if ( postType === NAVIGATION_POST_TYPE ) {
 		if ( postId ) {
```

Comment:
Nit: not from this PR but the only different thing we return in `if ( postType === NAVIGATION_POST_TYPE ) {` is the  backpath and we could apply the `if ( postId ) {` check there.

Reply:
The sidebar component used is different as well.